src/main/resources/changelog.xml

```diff
@@ -934,182 +986,236 @@
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint referencedTableName="repository_issues"
+                                 referencedColumnNames="repository_id, issue_id"
+                                 constraintName="repository_for_label"
+                                 baseTableName="issue_labels"
+                                 baseColumnNames="repository_id, issue_id"/>
+
+        <addForeignKeyConstraint referencedTableName="repository_labels"
+                                 referencedColumnNames="label_id"
+                                 constraintName="issue_for_label"
+                                 baseTableName="issue_labels"
+                                 baseColumnNames="label_id"/>
+    </changeSet>
+
+    <changeSet id="assign_ta_to_assignment" author="Maaike Visser">
+        <createTable tableName="assigned_ta">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="course_edition_id" type="bigint">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="group_number" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ta_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="assignment_course_edition_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="assignment_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint constraintName="course_edition_ref_in_assigned_ta"
+                                 baseTableName="assigned_ta" baseColumnNames="course_edition_id"
+                                 referencedTableName="groups" referencedColumnNames="course_edition_id"/>
+
+        <addForeignKeyConstraint constraintName="group_ref_in_assigned_ta"
+                                 baseTableName="assigned_ta" baseColumnNames="group_number"
+                                 referencedTableName="groups" referencedColumnNames="group_number"/>
+
+        <addForeignKeyConstraint constraintName="user_ref_in_assigned_ta"
+                                 baseTableName="assigned_ta" baseColumnNames="ta_id"
+                                 referencedTableName="users" referencedColumnNames="id"/>
+
+        <addForeignKeyConstraint constraintName="course_ref_in_assigned_ta"
+                                 baseTableName="assigned_ta" baseColumnNames="assignment_course_edition_id"
+                                 referencedTableName="assignments" referencedColumnNames="course_edition_id"/>
+
+        <addForeignKeyConstraint constraintName="assignment_ref_in_assigned_ta"
```

Comment:
Nice constraints! 👍 Could perhaps use a `unique constraint` on `course_edition_id`, `group_number` and `assignment_id` too (to prevent multiple TA's being assigned, or the same TA being assigned twice).